### ISSUE TITLE
qa: determinism + race witness for Qwen3.8 paged swap bulk codec

### ISSUE THREAD

**Issue by anthony-chaudhary** [anthony-chaudhary/fak#9616]
## Generation stream

gen/now

## Parent context

#8395

## Current state

The working spine of Qwen3.8 paged swap bulk codec exists (bc95ccc313b6c6d3c841f1b1e5bc0fe99aee70a7); this follow-on has not started.

## Why this is next

Spine-first default: the spine just shipped, so its follow-on backlog is filed at creation time while context is hot.

## Working spine

The same input yields byte-identical output every run, and the package is race-clean, so cohort/replay layers can trust it.

## Core through-line

A determinism test (two runs, deep-equal) and a -race run over internal/model/.

## Gold-plating boundary

Performance tuning.

## Problem frame

- Centrality: enabling (the shipped spine's observable Core outcome)
- P1: advanced - the child captures its own bounded follow-on context while reusing the shipped spine
- P2: advanced - the child closes a named operating or evidence gap instead of expanding the spine
- P3: preserved - the child stays independently dispatchable and can be reprioritized without changing sibling scope
- P4: advanced - the child carries its witness through the real follow-on planning and dispatch path

## Done condition

Determinism test lands; -race run over the package is clean.

## Witness

go test -race ./internal/model -run Determinism -v output captured in the commit or issue.

## Acceptance gate

go test -race ./internal/model -run Determinism -v and make test-race green.

## Work estimate

Estimate: 2 points

## Overall completion contribution

Contribution: 2/8 points

## Completion standard

production

## Target operating envelope

- serialized payload bytes: >= 158488532 bytes
- codec round-trip exactness: = 100 percent

## Witnessed operating envelope

- serialized payload bytes: 158488532 bytes
- codec round-trip exactness: 100 percent

## Closure binding

Close via a commit whose body carries `Closes #<n>` and whose diff lands the witness above.

## Lane

model

## Work unit

leaf

## Expected steps

2

## Assumptions

- The spine at bc95ccc313b6c6d3c841f1b1e5bc0fe99aee70a7 still builds/runs on trunk.

## Confusion risks

- Map iteration order and clock/rand reads are the usual determinism leaks.
- Harden what exists; do not grow the spine's scope inside this follow-on.

## Coordination

- Stay inside internal/model/; check the lane lease before dispatch.

## Trigger

one-shot: filed at spine-ship time by fak issue fanout

## Batch policy

one batch per spine, capped at --max candidates; marker-key (fanout-model-*) dedupe against existing issues before filing

## Model tier

Required model tier: T2
Optimal model tier: T1

## Likely files

- `internal/model/`

Filed by `fak issue fanout --live`; the marker key above is the rerun dedupe contract.

**Comment by anthony-chaudhary** [comment]
Closure evidence on authoritative origin/main:

- Duplicate search found no separate determinism ticket; #9616 is the sole matching issue.
- Failure-before witness: go test -race ./internal/model -run Determinism -v -count=1 returned PASS with [no tests to run], which is CLAIM_TEST_UNRUN and could not close the issue.
- Resolving commit: 66e73cec684e95ab02435b4c9bf86e14cb807f66 (internal/model@r534+g66e73cec6), DCO-signed, recognized (fak model), exact one-file test diff, pushed as origin/main. The body carries Closes #9616.
- The existing real-seam round-trip test is now selectable as TestQwenHybridPagedSwapRoundTripDeterminism. No codec or product behavior changed.
- Witness contents: two independent serializations are byte-identical; restored K/Kraw/V, convolution, and recurrent state are bit-identical; continuation logits remain bit-identical.
- Repeated focused gate: go test ./internal/model -run Determinism -count=20 — PASS.
- Post-push race gate: go test -race ./internal/model -run Determinism -v -count=5 — PASS; the test ran five times, so the done-binding is CLAIM_TEST_GREEN.
- Exact isolated validation: importer build, vet, and all eight owned paged-swap tests — PASS.
- Strict full package after #9756 landed: go test ./internal/model -count=1 — PASS in 68.301s.
- dos commit-audit 66e73cec6 — diff-witnessed test claim.

Promotion evidence: the exact acceptance selector now executes the real paged-swap round-trip/determinism witness under the race detector instead of vacuously passing.

Demotion/retirement evidence: retire this named selector only if a replacement deterministic codec witness remains directly selectable by the issue gate and preserves the same byte/state/continuation checks.

Invalidating assumption: reopen if -run Determinism again matches no test, repeated serialization differs, restored state or continuation logits drift, or the focused race run reports a race.

#9616 is complete with CLAIM_TEST_GREEN.
